### DISCUSSION TITLE
DragonOS Pi64 and Cariboulite

### DISCUSSION THREAD

**Discussion by hb9dug** [cariboulabs/cariboulite#122] [General]
Build Cariboulite of today (2f718aa) on DragonOS_Pi64_Beta30.img with no problems.
Gnu Radio and SDR++ working on receive using sudo. Still the issue of Pulseaudio.

Thank you David for your effort improving the software.

**Comment by meexmachina** [comment]
Thanks,  the sudo probelm should dissapear in a little while - fixing the udev issue of installation in Ubuntu.
